Source/Scene/PointCloud3DTileContent.js

```diff
@@ -1004,7 +1020,13 @@ define([
             vs += '    gl_PointSize = u_pointSize; \n';
         }
 
-        vs += '    color = color * u_highlightColor; \n';
+        vs += '    vec4 position_EC = czm_view * vec4(position_absolute, 1.0); \n' +
```

Comment:
For style purposes rename `position_EC` to `positionEC`. It may be worth renaming `position_absolution` to `positionWC` (except line 784 which relies on the name being the lowercase of the semantic name).